StencilFrontend.md

```diff
@@ -33,12 +33,11 @@ cd ~/FoodTrackerBackend/FoodServer
 mkdir Views
 cd Views
 ```
-3. Create a `FoodTemplate.stencil` file.
-We will use terminal commands for file creation but you can use a text editor of your choice.
+3. Create a `FoodTemplate.stencil` file, then open it with any text editor:
 ```
-cat > FoodTemplate.stencil
+touch FoodTemplate.stencil
+open FoodTemplate.stencil
```

Comment:
What happens if there's no filetype handler for `.stencil`?

Reply:
Good catch. Fixing with a flag for `-a` in the bash command.